### DISCUSSION TITLE
Style Manager: Individual border for mjml elements ( border left,border right, border bottom, border top)

### DISCUSSION THREAD

**Discussion by mkandanmk7** [GrapesJS/grapesjs#5171] [Q&A] [answered]
**Used packages Details:**
1.  "grapesjs": "^0.20.2",
2.  "grapesjs-mjml": "^1.0.3",

I trying get individual border for mjml element section **(mj-section)** 
while select mj-section i need to show individual border styles under decoration section,
This feauture is really important kind check and update. 

**Expected Result:** 
![image](https://github.com/GrapesJS/grapesjs/assets/84056393/2204b863-5f62-448f-a8ff-d2678e8e73e8)
**(Its used in old version )**
**Actual Result:** 
![image](https://github.com/GrapesJS/grapesjs/assets/84056393/addde92a-2552-4e36-899b-a1a510d047b6)

**Comment by artf** [comment]
Here you can find all the details for setting up the Style Manager: https://grapesjs.com/docs/modules/Style-manager.html

**Reply by mkandanmk7** [reply, marked as answer]
Thank you artf.  https://gist.github.com/mkandanmk7/8accda4816de472e9a42cc9888fe9a08 
here Initialy reset stylemanager sectors array, then added needed style properties inside the sectors array. I make it as plugin .People who need custom style manager use this . 
https://gist.github.com/mkandanmk7/8accda4816de472e9a42cc9888fe9a08

**Comment by mkandanmk7** [comment]
![image](https://github.com/GrapesJS/grapesjs/assets/84056393/720baab7-c9d6-46f0-82fb-db8eee4d5138)
